main/src/library/pkger/pkger.flix

```diff
@@ -0,0 +1,43 @@
+pub enum SemVerComponent {
+    case Integer(Int),
+    case Wildcard,
+    case NotPresent
+}
+
+pub opaque type SemVer = { major: SemVerComponent
+                         , minor: SemVerComponent
+                         , patch: SemVerComponent
+                         }
+
+// Temporary
+pub opaque type SpecificSemVer = SemVer
+
+pub opaque type Package = { name: String
+                          , version: SpecificSemVer
+                          , authors: List[String]
```

Comment:
I think we want an ADT for authors (e.g. first name, last name, email etc.)

Reply:
Good idea. I was just going by a git-style author string for this. But an ADT-based approach is better.

Reply:
After thinking a bit about it, I don't think breaking out the names into first- and last-names is something we want to do. Too many edge-cases for names. A single `name` string avoids most cultural clashes, and the user gets to choose how they're represented.

Reply:
Name and email then?

Reply:
Yes, that is what I've locally for now.